### ISSUE TITLE
Additional widget in the top screen of Tandem XES radio

### ISSUE THREAD

**Issue by fdapport** [FrSkyRC/ETHOS-Feedback-Community#4510]
Firmware: 1.5.15
Hardware: Tandem Frsky XES

It will be nice to have the possibility to set in the top small screen of the Tandem XES radio different widget as the bottom  screen. 
As now it is possible to set a bitmap, value, timer logs, timer value, GPS map, lipo, channels, line chart, text. 
It is possible to have the possibility to set a additional widget with 2 or 4 values. It is a enhancements for the current version
Thanks, Franco

**Comment by Dsrenger** [comment]
I am confused,

You can set up to 4 different values to the top screen of the XE. Not dependent on the bottom screen.

Isnt that, what you want?

![image](https://github.com/user-attachments/assets/8e578987-b277-4028-bebd-3f5474b32942)

**Comment by fdapport** [comment]
yes,, this is what I want... let me try again in the simulator If I could change it....

**Comment by Dsrenger** [comment]
In the screen configurator you need to double tap on "Screen Top" and then "change layout"

**Comment by fdapport** [comment]
Great... resolved and happy!!!  I close the ticket...Thanks, Franco
